react/components/EXPERIMENTAL_Table/README.md

```diff
@@ -642,6 +642,1091 @@ function VisibilityExample() {
 ;<VisibilityExample />
 ```
 
+# Features
+
+<div className="center mw7 pv6">
+  ![](./table.png)
+</div>
+
+## Toolbar
+
+The toolbar is a bundle of features, including search input, autocomplete, columns visibility toggler, density controls, import and export buttons, extra actions menu using ActionMenu component and a newLine button to help with entry creation (you can see the illustrative diagram in the begining of the page for a better visualization of this structure)
```

Comment:
Please, run the [grammarly](https://www.grammarly.com/) at the docs. There are some typos that can be fixed.